implementations/rust/ockam/ockam_api/src/nodes/models/portal.rs

```diff
@@ -150,11 +150,16 @@ impl CreateOutlet {
 #[rustfmt::skip]
 #[cbor(map)]
 pub struct InletStatus {
+	#[serde(rename = "listen_address")]
     #[n(1)] pub bind_addr: String,
+    #[serde(skip)]
     #[n(2)] pub worker_addr: String,
+    #[serde(skip)]
     #[n(3)] pub alias: String,
     /// An optional status payload
+    #[serde(skip)]
     #[n(4)] pub payload: Option<String>,
+	#[serde(rename = "route_to_outlet", serialize_with = "crate::util::serialize_route_as_multiaddr")]
     #[n(5)] pub outlet_route: String,
```

Comment:
We should revert the serde customizations for this struct and the OutletStatus. We use them in other crates "as is", so they rely on the current unmodified structure, and these changes would break that code.